src/main/resources/templates/add-category.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
hm, nie szłabym w tym kierunku. Zamast tego robiłabym testy + skrypty do uruchomienia czy to w kliencie w IntelliJ czy w zewnętrznym

Reply:
usuniete